windows/tools/compile_bitmap.py

```diff
@@ -127,8 +129,7 @@ def bitmap_from_bitmap_source(app, e=None):
             window.tag, width, height, 1, "texture_2d", "a8r8g8b8", 0, pixels)
 
         window.tag.rel_filepath = "untitled%s.bitmap" % app.untitled_num
-        window.tag.filepath = os.path.join(
-            app.tags_dir + window.tag.rel_filepath)
+        window.tag.filepath = app.tags_dir.joinpath(window.tag.rel_filepath)

```

Comment:
Please also just go for `Path(a, b)` here instead of `a.joinpath(b)`

Reply:
See above comment.